src/index.html

```diff
@@ -11,7 +11,161 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal Information</legend>
+        <label class="form_field">
+          Surname :
+          <input
+            name="Surname"
+            type="text"
+            autocomplete="off"
+            required
+            minlength="5"
+            maxlength="35"
+          >
+        </label>
+        <label class="form_field">
+          Name :
+          <input
+            name="Name"
+            type="text"
+            autocomplete="off"
+            required
+            minlength="5"
+            maxlength="25"
+          >
+        </label>
+        <label class="form_field">
+          How old are you?:
+          <input
+          name="Age"
+          id="Age"
+            type="number"
+            required
+            minlength="1"
+            maxlength="100"
```

Comment:
These attributes are redundant for input with number type. Use min and max instead
<img width="255" alt="image" src="https://github.com/mate-academy/layout_html-form/assets/99403198/30a02f51-85b7-482d-b384-e571bf348879">